workflowhelpers/internal/space.go

```diff
@@ -111,9 +120,16 @@ func (ts *TestSpace) Create() {
 }
 
 func (ts *TestSpace) Destroy() {
-	deleteOrg := internal.Cf(ts.CommandStarter, "delete-org", "-f", ts.organizationName)
-	EventuallyWithOffset(1, deleteOrg, ts.Timeout).Should(Exit(0))
-
+	if ts.isExistingOrganization {
+		setDefaultQuota := internal.Cf(ts.CommandStarter, "set-quota", ts.organizationName, "default")
```

Comment:
The quota deletion will fail if the quota is still associated with an org.

The line assumes that the previous quota on the existing organization was `default` which may not always be true.